OpenRA.Mods.Common/Projectiles/Bullet.cs

```diff
@@ -110,8 +110,11 @@ public class BulletInfo : IProjectileInfo
 		[Desc("Equivalent to sequence ZOffset. Controls Z sorting.")]
 		public readonly int ContrailZOffset = 2047;
 
-		[Desc("Thickness of the emitted line.")]
-		public readonly WDist ContrailWidth = new WDist(64);
+		[Desc("Thickness of the emitted line at the start of the contrail.")]
+		public readonly WDist ContrailStartWidth = new WDist(64);
+
+		[Desc("Thickness of the emitted line at the end of the contrail. Set to StartWidth if undefined")]
```

Comment:
Use `nameof(ContrailStartWidth)` in the description.
"Will default to ... if left undefined."